spec/unit/api_spec.rb

```diff
@@ -114,6 +114,30 @@
     end
   end
 
+  describe ".parse_environmentdir" do
+
+    it "calls parse_deployed_env on each dir in the path" do
+      @env_path = '/tmp/environments'
```

Comment:
Using instance variables to carry state between tests can lead to order specific test failures; prefer `let` blocks (http://betterspecs.org/#let) and initializing objects with `before` blocks over instance variables.